lesion_segmentation_protocol.md

```diff
@@ -0,0 +1,31 @@
+# Lesion segmentation protocol:
+
+The following details the protocol for Multiple Sclerosis (MS) lesion segmentation in the spinal cord.
+Imaging the spinal cord is often essential to confirm the diagnosis of MS. That is because the lesions of the spinal cord are included in the McDonald diagnosis criteria, which studies dissemination in space and in time [(Thompson et al. 2018)](https://pubmed.ncbi.nlm.nih.gov/29275977/). While the MAGNIMS-CMSC-NAIMS working group recommends to use at least two sagittal images for MS diagnosis, still, axial imaging is mentioned as optional in international imaging guidelines [(Wattjes et al. 2021)](https://pubmed.ncbi.nlm.nih.gov/34139157/).
+For detecting MS lesions in the spinal cord, two main contrasts emerge: PSIR and STIR contrasts. New studies [(Peters et al. 2024)](https://pubmed.ncbi.nlm.nih.gov/38289376/)[(Fechner et al. 2019)](https://pubmed.ncbi.nlm.nih.gov/30679225/) showed that using PSIR contrasts improved MS lesion detection in the spinal cord. [(Fechner et al. 2019)](https://pubmed.ncbi.nlm.nih.gov/30679225/) showed that the PSIR contrast showed a higher signal-to-noise (SNR) ratio compared to the STIR contrast. 
+
+## Criteria to segment MS lesions in the spinal cord:
+
+Do not segment lesions in images with too many artifacts (such as this one : https://github.com/ivadomed/canproco/issues/53#issue-1938136790). Preferably, add the image to the exclude file so that it isn’t used for model training…
```

Comment:
Question : in words, what would be an image considered being too artefacted? Any criteria?
```suggestion
Do not segment lesions in images with too many artifacts (such as this one : https://github.com/ivadomed/canproco/issues/53#issue-1938136790). Preferably, add the image to the exclude file so that it isn’t used for model training…
```